docs/user-manual/index.md

```diff
@@ -914,6 +914,15 @@ configuration or user group management.
 ./user-sync –c user-sync-config.yml --users groups "group1, group2, group3"
 ```
 
+#### Sync only users in given groups
+
+This action is the same as specifying --users groups "...", but adds all
```

Comment:
Be sure to use backticks around `--users groups`.  Same comment as above, please reword this to make clear that there is no "..." argument.  Also no but.  Also there is only one configuration file where the group mapping exists.